config.go

```diff
@@ -393,7 +410,7 @@ func loadConfig() (*config, []string, error) {
 		// Created successfully, so exit now with success.
 		os.Exit(0)
 
-	} else if !fileExists(dbPath) {
+	} else if !fileExists(dbPath) && cfg.CreateTemp == false {
```

Comment:
I'd reorganize this to simplify the conditions by checking the temp check first.  Also, no need to use `== false/true` as `if cfg.CreateTemp` or `if !cfg.CreateTemp` is the idiomatic approach.

```Go
if cfg.CreateTemp {
	// Do the temp creation bits
} else if cfg.Create {
	// Do the normal creation bits
} else if !fileExists(dbPath) {
	// Error bits.
}
```